modules/contentbox/modules/contentbox-admin/handlers/authors.cfc

```diff
@@ -438,6 +444,10 @@ component extends="baseHandler"{
 		authorService.saveAuthor( oAuthor );
 		// message
 		cbMessagebox.setMessage( "info","Two Factor Settings Saved!" );
+		if ( !prc.oCurrentAuthor.isLoaded() ){
```

Comment:
I am not sure why you are doing this? What is the point of this?

Reply:
Not sure about this. That code was already there, I am afraid of taking it out. It might mess up something that is using the author stored in the session.

Reply:
Let me rephrase, why log in the user?